packages/curriculum-compiler-json/compilers/helpers.js

```diff
@@ -0,0 +1,36 @@
+const { getCompiler } = require('@enkidevs/curriculum-compiler-string')
+const { contentTypes } = require('@enkidevs/curriculum-helpers')
+
+function getInsightCompiler () {
+  return getCompiler(contentTypes.INSIGHT)
+}
+function compileNodeToInsightMarkdown (node) {
+  if (!node || !node.children) {
+    throw new Error('Cannot compile invalid node')
+  }
+  return getInsightCompiler().compileSync({
```

Comment:
Just a note, you can use [`unist-builder`](https://www.npmjs.com/package/unist-builder) for node creation